README.rst

```diff
@@ -141,6 +142,12 @@ Containers`_.
     >>> help(SortedDict)
     >>> help(SortedDict.popitem)
 
+If you'd like type annotations, they're available by installing the
+`sortedcontainers-stubs <https://pypi.org/project/sortedcontainers-stubs/>`_
+package (maintained separately, by Sorted Container's users)::
```

Comment:
I mainly mention it's maintained separately because the trust assumptions are different for the stubs package, and people could think it was part of this repo. Also it might avoid people opening issues here for type problems!